controllers/datafoundationclient.go

```diff
@@ -40,11 +43,16 @@ const (
 	csiKMSConnectionDetailsConfigMapName = "csi-kms-connection-details"
 	storageClientCRDName                 = "storageclients.ocs.openshift.io"
 	storageClassClaimCRDName             = "storageclassclaims.ocs.openshift.io"
+	defaultStorageClientName             = "ocs-storageclient"
```

Comment:
Why default? this is the only storageclient we support right now. 
```suggestion
	storageClientName             = "ocs-storageclient"
```

Reply:
updated